core/src/trezor/ui/components/common/confirm.py

```diff
@@ -1,11 +1,9 @@
 from typing import TYPE_CHECKING
```

Comment:
can't we remove this whole file? I was under the distinct impression that I removed it [here](https://github.com/trezor/trezor-firmware/pull/2556) but apparently not...?

Reply:
We certainly can, after the deletion, it contains just 
```
CONFIRMED
CANCELLED
INFO
def is_confirmed(x: Any) -> bool:
```
which each module could do on its own. We will get rid of one nested directory in a file structure

Reply:
which is the case for tt_v2 already anyway

Reply:
Done in [4bc7990](https://github.com/trezor/trezor-firmware/pull/2688/commits/4bc79909321a8d407551645b22f1043266507bbe)



Turned out there are two places that used it:
```
core/src/apps/bitcoin/sign_tx/approvers.py:215: - error: Import "trezor.ui.components.common.confirm" could not be resolved (reportMissingImports)

core/src/trezor/ui/layouts/tt_v2/__init__.py:549: - error: "confirm" is unknown import symbol (reportGeneralTypeIssues)
```

I solved it (little dirty) by sending a boolean saying if it was confirmed (True) of info was requested (False)